README.md

```diff
@@ -4,7 +4,7 @@
 
 Automated web accessibility testing with .NET, C#, and Selenium. Wraps the [axe-core](https://github.com/dequelabs/axe-core) accessibility scanning engine and the [Selenium.WebDriver](https://www.seleniumhq.org/) browser automation framework.
 
-Compatible with .NET Standard 2.0+, .NET Framework 4.5+, and .NET Core 2.0+.
+Compatible with .NET Standard 2.0+, .NET Framework 4.5+, and .NET Core 3.1+.
```

Comment:
Is this change intentional? I think the original is still more about, since it doesn't look like you're removing .NET Core [2.0, 3.1) support

Reply:
You are correct